### DISCUSSION TITLE
Welcome to ACRE2 Discussions!

### DISCUSSION THREAD

**Discussion by jonpas** [IDI-Systems/acre2#1083] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Simonusios** [comment]
Hey all! I am Simon, a mad mission maker, and I had a fun idea that I don't know how to actually implement.

I would like to be able to mute all the players, but locally. Think hideObject vs hideObectGLobal. One player should not be able to hear any other players.
Why? I am making a little pocket dimension where a player might get pulled into after touching things they are not supposed to touch. I want to make them feel isolated, but I can't teleport them away from the other players. Thus I am making the other players invisible to the target player.
The only downside is that they would still be able to hear the others and talk to them (although setting their global volume is an easy fix for that last one)

I have looked through the documentation and have seen these two functions:

acre_api_fnc_setGlobalVolume
acre_api_fnc_isSpeaking (could be used to determine when the volume should be changed)

But setGlobalVolume seems to be global. This means if I were to call it on the players that will turn invisble to the targeted player the other players would not be able to talk anymore either. This is not optimal.

Thus the question: 
Is there a way to change the player volume locally? Or is there some workaround I havent thought of?

Thank you for your time!
Kind regards,
Simon
